src/lib/router.ts

```diff
@@ -235,8 +229,8 @@ export function routeToPath(route: Route): string {
     return seedPath(route.params.baseUrl);
   } else if (route.resource === "loadError") {
     return "";
-  } else if (route.resource === "projects") {
-    return projectRouteToPath(route.params);
+  } else if ("project" in route) {
```

Comment:
Maybe add a comment explaining how this works.

Reply:
Instead of explaining I chose the more straightforward but more verbose implementation: d751e7c09a0d8513126d7b8c6cdc4bd9309dcf79